lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+  attr_reader :name, :age, :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {
+      written: false,
+      license: false,
+      renewed: false
+    }
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
```

Comment:
Nice job here. You could add a guard clause like
```ruby
@permit = true if @age >= 15
```
To ensure that only eligible registrants can get permits.